packages/web-config-server/src/apiV1/DataAggregatingRouteHandler.js

```diff
@@ -59,12 +50,36 @@ export class DataAggregatingRouteHandler extends RouteHandler {
       (obj, countryCode, i) => ({ ...obj, [countryCode]: countryAccessList[i] }),
       {},
     );
-    dataSourceEntities = dataSourceEntities.filter(e => countryAccess[e.country_code]);
+    const permittedDataSourceEntities = dataSourceEntities.filter(
```

Comment:
Was there some advantage to mutating `dataSourceEntities`? This is only 1 line longer and I personally find it easier to read.